populator-machinery/controller_test.go

```diff
@@ -61,16 +62,25 @@ type testCase struct {
 	populateFn func(context.Context, PopulatorParams) error
 	// Provider specific data population completeness check function, return true when data transfer gets completed.
 	populateCompleteFn func(context.Context, PopulatorParams) (bool, error)
+	// PvcPrimeMutator is the mutator function for pvcPrime
+	pvcPrimeMutator func(MutatorParams) error
 	// Expected errors
 	expectedResult error
 	// Expected keys in the notifyMap
 	expectedKeys []string
 	// Expected objects after the test runs
 	expectedObjects []runtime.Object
+	// Expected objects after modification
+	expectedMutateObjects mutateObjects
```

Comment:
s/expectedMutateObjects/expectedMutatedObjects

Reply:
Change from `expectedMutateObjects mutateObjects` to `expectedObjects vpObjects` to avoid confusion for tests which don't test mutators as the suggestion below